tests/functional/aws-node-sdk/test/multipleBackend/objectPutCopyPart/objectPutCopyPartAzure.js

```diff
@@ -675,3 +676,131 @@ function describeF() {
+                this.currentTest.mpuKeyNameAzure =
+                `mpukeyname${uniqName(keyObjectAzure)}`;
+
+                const params = {
+                    Bucket: azureContainerName,
+                    Key: this.currentTest.mpuKeyNameAzure,
+                    Metadata: { 'scal-location-constraint': azureLocation },
+                };
+                async.waterfall([
+                    next => s3.createBucket({ Bucket: awsBucketName },
+                      err => next(err)),
+                    next => s3.createBucket({ Bucket: azureContainerName },
+                      err => next(err)),
+                    next => s3.putObject({
+                        Bucket: awsBucketName,
+                        Key: this.currentTest.keyNameAws,
+                        Body: fiveMbBody,
+                        Metadata: { 'scal-location-constraint': awsLocation },
+                    }, err => next(err)),
+                    next => s3.createMultipartUpload(params, (err, res) => {
+                        assert.equal(err, null, 'createMultipartUpload: ' +
+                        `Expected success, got error: ${err}`);
+                        this.currentTest.uploadId = res.UploadId;
+                        next();
+                    }),
+                ], done);
+            });
+
+            it('should copy two 5 MB part from Azure to MPU with Azure ' +
+            'location', function ifF(done) {
+                const params = {
+                    Bucket: azureContainerName,
+                    CopySource:
+                      `${awsBucketName}/` +
+                      `${this.test.keyNameAws}`,
+                    Key: this.test.mpuKeyNameAzure,
+                    PartNumber: 1,
+                    UploadId: this.test.uploadId,
+                };
+                const params2 = {
+                    Bucket: azureContainerName,
+                    CopySource:
+                      `${awsBucketName}/` +
+                      `${this.test.keyNameAws}`,
+                    Key: this.test.mpuKeyNameAzure,
+                    PartNumber: 2,
+                    UploadId: this.test.uploadId,
+                };
+                async.waterfall([
+                    next => s3.uploadPartCopy(params, (err, res) => {
+                        assert.equal(err, null, 'uploadPartCopy: Expected ' +
+                        `success, got error: ${err}`);
+                        assert.strictEqual(res.ETag, `"${fiveMbMD5}"`);
+                        next(err);
+                    }),
+                    next => s3.uploadPartCopy(params2, (err, res) => {
+                        assert.equal(err, null, 'uploadPartCopy: Expected ' +
+                        `success, got error: ${err}`);
+                        assert.strictEqual(res.ETag, `"${fiveMbMD5}"`);
+                        next(err);
+                    }),
+                    next => {
+                        const params = {
```

Comment:
you should define this with the other params (params & params2) if possible and call it `completeMpuParams` (you can call the other params `uploadParams` & `uploadParams2`) or just `params3` to avoid the overlapping naming scopes